tests/cpp/test_gpu_view.cpp

```diff
@@ -2317,4 +2317,86 @@ TEST_F(GpuViewTest, ExpandedBroadcast) {
   testValidate(&fusion, {actual_out_tensor}, {in_tensor}, __LINE__, __FILE__);
 }
 
+using ViewReductionParamType = std::vector<int64_t>;
+class ViewReductionParametrizedTest
```

Comment:
Not a change request, just FYI: you can use the same parameterized test fixture for both TEST_F and TEST_P. E.g. https://github.com/NVIDIA/Fuser/blob/7bc7a08dac0dc7473bc34fd2280b918d82e39ae8/tests/cpp/test_memory.cpp#L45 and https://github.com/NVIDIA/Fuser/blob/7bc7a08dac0dc7473bc34fd2280b918d82e39ae8/tests/cpp/test_memory.cpp#L117

However, in this case, the parameterized test fixture is specific to view reduction, so the change as is is actually better.